stripe/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -165,6 +207,10 @@ class Stripe internal constructor(
      */
     @JvmOverloads
     @UiThread
+    @Deprecated(
+        "ComponentActivity will be required in an upcoming major version.",
+        ReplaceWith("confirmPayment(ComponentActivity)")
```

Comment:
I'm assuming we want the user to make the necessary adjustments here instead of providing a hands-off replacement. This makes sense to me because we can't blindly cast the activity instance as a `ComponentActivity`.

Reply:
Yeah, this is perhaps a non-standard usage of `ReplaceWith`, but I think it will help clarify the change.